api/src/pcapi/scripts/payment/mark_payments_as_sent.py

```diff
@@ -43,14 +53,63 @@ def mark_payments_as_sent(transaction_label: str, batch_size: int = 1000) -> Non
         if len(payments_ids) == 0:
             continue
 
-        payment_statuses_to_add: list[PaymentStatus] = []
-        for payment_id in payments_ids:
-            payment_statuses_to_add.append(PaymentStatus(paymentId=payment_id, status=TransactionStatus.SENT, date=now))
-
-        db.session.bulk_save_objects(payment_statuses_to_add)
-        mark_bookings_as_reimbursed_from_payment_ids(payments_ids, now)
-        db.session.commit()
-
+        mark_payments_batch_as_sent(payments_ids, now)
         modified_sum += len(payments_ids)
 
     logger.info("%d payments have been marked as sent for transaction %s", modified_sum, transaction_label)
+
+
+def mark_payments_as_sent_from_file(ids_file: str, transaction_label: str, batch_size: int = 1000) -> None:
+    """
+    Allows to mark as SENT payments whose ids are read from a text file.
+    The transaction label only to check the payments' ones.
+    The file must be a text file with one id per line.
+
+    ---usage---
+
+    Copy the text file on the pod before running the function with `kubectl ps`:
+        kubens staging  # choose the correct namespace
+        kubectl get pods  # eventually list the pods to retrieve the wanted one (console)
+        kubectl cp /local/path/to/reimbursement_ids.txt <the console pod name>:/usr/src
+
+    Open a python interpreter on the pod:
+        pc -e staging python
+
+    Run the function:
+        from pcapi.scripts.payment.mark_payments_as_sent import mark_payments_as_sent_from_file
+        mark_payments_as_sent_from_file('/usr/src/reimbursement_ids.txt', '<the related transaction label>')
+
+    """
+    batches = []
+    with open(ids_file, "r") as reimbursement_ids:
+        batch_remain = batch_size
+        batch = []
+        for reimbursement_id in reimbursement_ids:
+            batch.append(int(reimbursement_id))
+            batch_remain -= 1
+
+            if not batch_remain:
+                batches.append(batch)
+                batch = []
+                batch_remain = batch_size
+
+        if batch:
+            batches.append(batch)
```

Comment:
(détail / style / question de goût) Variante : 

```suggestion
    with open(ids_file, "r") as fp:
        ids = [int(_id.strip()) for _id in fp.readlines()]
    batches = [ids[index : index + batch_size] for index in range(0, len(ids), batch_size)]
```

Reply:
effectivement c'est un peu plus concis et pas moins lisible alors j'approuve à quelques modif près:
- j'utilise pas readlines car c'est pas nécessaire il me semble
- je rajoute un `if _id` dans la compréhension au cas ou on croise une ligne vide lors de la lecture du fichier